distribution/build.gradle

```diff
@@ -43,6 +46,21 @@ buildscript {
 Collection distributions = project.subprojects.findAll {
    it.path.contains(':tools') == false && it.path.contains(':bwc') == false }
 
+/*****************************************************************************
+ *                  Third party dependencies report                          *
+ *****************************************************************************/
+
+// Concatenates the dependencies CSV files into a single file
+task generateDependenciesReport(type: ConcatFilesTask) {
+  files = fileTree(dir: project.rootDir,
+          excludes: ['qa/**/dependencies.csv', 'benchmark/**/dependencies.csv', 'test/**/dependencies.csv'],
+          include: '**/dependencies.csv' )
+  headerLine = "name,version,url,license"
+
+  def outputFile = System.getProperty('csv')?: "${project.buildDir}/dependencies/es-dependencies.csv"
```

Comment:
We try not to use `def` anywhere. Also, why do we need a sysprop at all? We should always write to to the build tree.

Reply:
The `csv` property has been added to be consistent with other projects in the stack. It gives the possibility to generate dependencies reports of the entire Stack into a common target path.